doc/tutorials/videoio/intelperc.markdown

```diff
@@ -5,20 +5,17 @@ Using Creative Senz3D and other Intel Perceptual Computing SDK compatible depth
 
 **Note**: this tutorial is partially obsolete since PerC SDK has been replaced with RealSense SDK
 
-Depth sensors compatible with Intel Perceptual Computing SDK are supported through VideoCapture
+Depth sensors compatible with Intel® RealSense SDK are supported through VideoCapture
```

Comment:
Please update page title on line 1 too (but keep old `id`)

Reply:
Sorry, I didn't get the title.

Done